apps/api/src/services/tag/tag.service.ts

```diff
@@ -17,18 +18,33 @@ export class TagService {
     });
   }
 
-  public async getInUseByUser(userId: string) {
-    return this.prismaService.tag.findMany({
+  public async getTagsForUser(userId: string) {
+    const tags = await this.prismaService.tag.findMany({
       orderBy: {
         name: 'asc'
       },
+      include: {
+        _count: {
+          select: {
+            orders: true
```

Comment:
The specification was too unspecific. It needs the flag `isUsed` related to the current user:

```
orders: {
  where: {
     userId
  }
}
```

Reply:
3ed6027 - hmm, that makes sense, my bad 🤝